### ISSUE TITLE
hsd-client is not in the npm registry

### ISSUE THREAD

**Issue by foobar94** [handshake-org/handshake-org.github.io#11]
The documentation says "Install hsd-cli and hsw-cli command line tools with the hs-client package. Included with hsd by default, but can be installed separately:` npm install -g hsd-client`"

However, `npm install -g hsd-client`gives:
```
npm ERR! Linux 4.15.0-33-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "-g" "hsd-client"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2
npm ERR! code E404

npm ERR! 404 Not found : hsd-client
npm ERR! 404 
npm ERR! 404  'hsd-client' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

**Comment by kilpatty** [comment]
@foobar94 It's in the NPM registry as `hs-client`

See here: https://www.npmjs.com/package/hs-client

Going to push up a P.R. to fix this.

**Comment by boymanjor** [comment]
Daaaamn @kilpatty, back at it again with the dope comments lol. I didn't see a PR so I just merged a quick change fea3a7c223d25f3faf180730636a230d39a1cad1.
